pkg/maps/authmap/cell.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package authmap
+
+import (
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+// Cell provides the auth.Map which is responsible for interacting with the eBPF map auth.
+var Cell = cell.Module(
+	"auth-map",
+	"eBPF map which manages authenticated connections between identities",
+
+	cell.Provide(newAuthMap),
+	cell.Invoke(func(Map) {}),
```

Comment:
Do we need the forced instantiation?

Reply:
@joamaki 

no not really - having the auth manager depending on it - which triggers the instantiation.

but i thought it would be best to instantiate the map properly so the datapath can rely on it. otherwise it kind of always needs someone who depends on it to get initialized. or does datapath code create the map as well and would work without this initialization from userspace?

Reply:
@joamaki 

i removed the forced instantiation. because of the dependencies `daemon` -> `auth manager` -> `auth map` the initialization should be triggered anyway and before the loader gets initialized.